Cartfile

```diff
@@ -1,7 +1,7 @@
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.2
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 5.6
-binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 6.2.1
-github "mapbox/mapbox-directions-swift" ~> 0.31
+binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 9.0.3
+github "mapbox/mapbox-directions-swift" == 1.0.0-alpha.1
```

Comment:
This should’ve remained v0.31. Looks like a rebase incorrectly changed it back to v1.0.0-alpha.1. Fixed in c3dda0043f8a933668b44cfca997daa2e0e37c2d.